content/api/power-board.md

```diff
@@ -23,3 +23,30 @@ r.power_board.power_on()
 {{% notice warning %}}
 Because the motor board is powered through these power outputs, whilst the power is off, you won't be able to control your motors. This will register as a missing board and your code will break if you try and control them.
 {{% /notice %}}
+
+## Buzzing :bee:
+The power board has a piezo buzzer which can buzz.
+
+The `buzz` function accepts multiple parameters, depending on what you want to play. The first argument is the duration of the beep, in seconds. The later arguments are either the note you want to play, or the frequency of the buzzer (in Hz). You have to specify which of note or frequency you're passing using a keyword argument, your code will fail otherwise.
+
+Theoretically, the piezo buzzer will buzz at any provided frequency, however humans can only hear between [20Hz and 20000Hz](https://en.wikipedia.org/wiki/Hearing_range#Humans).
+
+{{% notice tip %}}
+Calling `buzz` is non-blocking. Meaning it doesn't actually wait for the piezo to stop buzzing before continuing with your code. If you want duration to wait for the buzzing to stop, add a `sleep` afterwards!
+{{% /notice %}}
+
+
+```python
+# Buzz for half a second in D.
+r.power_board.buzz(0.5, note='d')
+
+# Buzz for 2 seconds at 400Hz
+r.power_board.buzz(2, frequency=400)
+
+# Will fail!
+r.power_board.buzz(2, 400)
+```
+
+{{% notice warning %}}
+Passing an invalid note will raise a `KeyError`. You can see all the supported notes by looking at the power boards `BUZZ_NOTES` variable: `print(r.power_board.BUZZ_NOTES)`
```

Comment:
Can we list these in the docs somewhere too? As it stands, users have to put this code snippet onto a USB stick, boot the robot up and examine the log in order to see what notes they can use with this function.

Reply:
I dont like documenting things like that, as they end up out of sync. Directly linking to the code isnt really ideal either

Reply:
Directly linking to the code isn't too bad IMO, as long as the code is clear (which in this case I feel it is - it's just a list of values). It also encourages users to dig around our codebases, which is probably a good thing.

Reply:
GitHub only allows linking to line nunmbers, so again there's a place it can get out of sync

Reply:
We're already asking them to look for the `BUZZ_NOTES` variable. If it's an obvious-looking array near the top of the file then it should be easy for people to find.

Reply:
Hmm, i guess just linking to the file might work too

Reply:
Yes, the docs can end up out of sync; that's the perennial downfall of manually written docs. Linking to code or suggesting that users print a variable for a list of valid values are unfriendly, even for experienced developers. I'm strongly in favour of listing this sort of thing in the docs.